packages/api/cms-api/src/access-log/access-log.interceptor.ts

```diff
@@ -0,0 +1,59 @@
+import { CallHandler, ExecutionContext, Injectable, Logger, NestInterceptor } from "@nestjs/common";
+import { GqlExecutionContext } from "@nestjs/graphql";
+
+/** TODOs
+ * Docs are missing
+ * Api password check
+ */
+const DAM_IMAGE_PATH = "/dam/images/:hash/:fileId";
+const DAM_FILE_PATH = "/dam/files/:hash/:fileId";
+
+@Injectable()
+export class AccessLoggerInterceptor implements NestInterceptor {
+    protected readonly logger = new Logger(AccessLoggerInterceptor.name);
+
+    intercept(context: ExecutionContext, next: CallHandler) {
+        const requestType = context.getType().toString();
+
+        let message = "";
+        const additionalRequestData = [];
+        let ignored = false;
+
+        if (requestType === "graphql") {
+            const graphqlExecutionContext = GqlExecutionContext.create(context);
+            const graphqlContext = graphqlExecutionContext.getContext();
+
+            additionalRequestData.push(`ip: ${graphqlContext.req.ip}`);
+            const user = graphqlContext.req.user;
+            if (user) additionalRequestData.push(`user: ${user.id} (${user.name})`);
```

Comment:
This is the same for GraphQL and HTTP, could you extract it into a private function?